### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN046.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#95] [General]
# https://blog.3place.cn/3place/lmyn/LMYN046.html

https://blog.3place.cn/3place/lmyn/LMYN046.html

**Comment by picklecai** [comment]
“儿子们都被杀死了，
底提真是万分难过。
罗摩！她于是说了话，
对丈夫摩哩遮迦叶波： （1.45.1）

’尊者！我是个儿子被杀的人，
被你那些强壮的儿子们杀死；
我希望长期苦行求得儿子，
这个儿子能够杀死天帝释。 （1.45.2）

这里对丈夫说：你那些强壮的儿子。
但前文说了，是因陀罗（就是天帝释）的儿子们杀死了她儿子们。
而且她此刻的请求是：希望丈夫再给她一个儿子，以能够杀死天帝释。

所以“你那些”强壮的儿子，错了吧？

**Comment by picklecai** [comment]
天帝释好可怕。甘愿做小伏低990年，到了最后抓住机会一击而胜。

**Comment by picklecai** [comment]
天帝释是个心机boy。做小伏低伺候底提990年。最后趁她睡着了，将可能杀死自己的生命打碎。
